### PR TITLE
combination of input lantency and frame throughput generator

### PR DESCRIPTION
**Pull request by ypwalter**
1. generator and its configuration file created
2. modified origin input latency generator
3. adding ilft gsearch cat case
4. modified all ail test cases so that they are using sample1
5. adding new ipynb for new generator "ilft"
6. modified image calculation position due to google image changes
7. keep event_time for ft & il in different variable
8. updating running_time_result to cover il and ft time frame

### CODE REVIEW

**Comment by ypwalter** [comment]
@askeing @ShakoHo @MikeLien r?

**Comment by ypwalter** [comment]
@askeing @ShakoHo @MikeLien r?

File: lib/generator/baseGenerator.py [hunk 1]
```diff
@@ -86,6 +86,33 @@ def init_result_dict_variable(self, default_number_list, default_list_list):
             init_result_dict[init_list_key] = []
         return init_result_dict
 
+    def generate_update_result_for_combination(self, input_update_result, input_compare_result, input_run_time_dict):
```
Comment (askeing) [inline lib/generator/baseGenerator.py]: The implementation of `generate_update_result_for_combination()` method should be put in sub-class `InputLatencyFrameThroughputGenerator`.

File: lib/sikuli/gsearch.sikuli/gsearch.py [hunk 2]
```diff
@@ -40,18 +40,16 @@ def focus_search_inputbox(self):
     # this calculation is based on english version of google image search only
     def click_result_image(self, x, y):
         a = -120 + 280 * (x - 1)
-        b = 275 + 200 * (y - 1)
+        b = 190 + 200 * (y - 1)
```
Comment (askeing) [inline lib/sikuli/gsearch.sikuli/gsearch.py]: IMO, we should open an issue for tracking the x, y axis issues of Google Images Search.
Currently, it's okay for this PR.

Comment (ypwalter) [inline lib/sikuli/gsearch.sikuli/gsearch.py]: Thanks. #757 is opened for this.

Comment (askeing) [inline lib/sikuli/gsearch.sikuli/gsearch.py]: 👍

File: lib/generator/baseGenerator.py (hunk 1 above)
Comment (ypwalter) [inline lib/generator/baseGenerator.py]: Done. Thanks

**Comment by ypwalter** [comment]
Thanks.

**Pull request merged**